src/base/bittorrent/torrenthandle.cpp

```diff
@@ -366,6 +366,27 @@ QString TorrentHandle::nativeActualSavePath() const
     return QString::fromStdString(m_nativeStatus.save_path);
 }
 
+bool TorrentHandle::isAutoManaged() const
+{
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    return m_nativeStatus.auto_managed;
+#else
+    return (m_nativeHandle.flags() & libt::torrent_flags::auto_managed);
```

Comment:
You should use `m_nativeStatus.flags` everywhere `m_nativeStatus` fields were used before.

Reply:
of course! done.